modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -845,6 +845,14 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     return false;
   }
 
+  /**
+   * Indicates whether a coin defaults to native segwit change outputs
+   * @returns {boolean}
+   */
+  defaultsToP2wshChange() {
```

Comment:
Maybe call this `defaultChangeAddressType()` which returns an element of some `UtxoAddressType` enum? I'm not sure if we already have an enum like that, but if not I'd be fine with defining one.